src/EnergyPlus/MixedAir.cc

```diff
@@ -3204,6 +3204,9 @@ namespace MixedAir {
 		// without AirloopHVAC objects).
 		if ( InitOAControllerSetUpAirLoopHVACVariables ) {
 			if ( NumPrimaryAirSys > 0 ) {
+				if ( !allocated( AirLoopZoneInfo ) ){
+					AirLoopZoneInfo.allocate( NumPrimaryAirSys );
```

Comment:
Because standalone ERV does all sorts of shenanigans with OA controllers before the airloops are set up. I've tried several solutions today but no success yet. There's way too many dependencies in this module between the various OA objects.

<div>-------- Original message --------</div><div>From: Richard Raustad <notifications@github.com> </div><div>Date:03/16/2016  8:36 PM  (GMT-06:00) </div><div>To: NREL/EnergyPlus <EnergyPlus@noreply.github.com> </div><div>Cc: "Michael J. Witte" <mjwitte@gard.com> </div><div>Subject: Re: [EnergyPlus] Fixed crash of having both zone and system ERV in
  user file (#5483) </div><div>
</div>In src/EnergyPlus/MixedAir.cc:

> @@ -3204,6 +3204,9 @@ namespace MixedAir {
>  		// without AirloopHVAC objects).
>  		if ( InitOAControllerSetUpAirLoopHVACVariables ) {
>  			if ( NumPrimaryAirSys > 0 ) {
> +				if ( !allocated( AirLoopZoneInfo ) ){
> +					AirLoopZoneInfo.allocate( NumPrimaryAirSys );
This array is allocated at line 1524 in GetOAControllerInputs. This code is in InitOAController. So why would this code get called before this array is allocated??

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub